code/modules/clothing/under/accessories/accessory.dm

```diff
@@ -814,6 +814,48 @@
 	item_state = "corset_blue"
 	item_color = "corset_blue"
 
+/obj/item/clothing/accessory/pride_pin
```

Comment:
Can you kindly make a main pin item with the core functionality at a higher level to allow additional pins in future? The pride pin can then be a subtype of the pin

Reply:
Can do! That's a great idea, people have expressed wanting lore pins.